tests/integration/refextract/test_refextract_tasks.py

```diff
@@ -39,7 +39,16 @@ def test_create_journal_kb_file(app):
         create_journal_kb_file()
 
     with open(path, 'r') as fd:
-        assert 'JOURNAL OF HIGH ENERGY PHYSICS---JHEP' in fd.read()
+        journal_kb = fd.read()
```

Comment:
`read`->`readlines`? that would make the assertions stronger.

Reply:
Good idea :+1:

Reply:
In the end I went for `.read().splitlines()` since it strips whitespace and I can keep the assertions a tiny bit cleaner.